specification/cognitiveservices/data-plane/Face/v1.0/Face.json

```diff
@@ -2506,6 +2851,36 @@
           "description": "Show failure message when training failed (omitted when training succeed)."
         }
       }
+    },
+    "ConfidenceProperty": {
+      "description": "A number ranging from 0 to 1 indicating a level of confidence associated with a property.",
+      "type": "number",
+      "minimum": 0,
+      "maximum": 1
+    }
+  },
+  "parameters": {
+    "ImageStream": {
+      "name": "Image",
+      "in": "body",
+      "required": true,
+      "x-ms-parameter-location": "method",
+      "description": "An image stream.",
+      "schema": {
+        "type": "object",
+        "format": "file"
+      }
+    },
+    "ImageUrl": {
```

Comment:
Shouldn't be this referenced from Common instead?
https://github.com/Azure/azure-rest-api-specs/blob/652cebd100c2483b7f63979ac0149ef3de6e6fea/specification/cognitiveservices/data-plane/Common/Parameters.json#L21